benchmark/make-project/server-stress-default.js

```diff
@@ -25,13 +27,22 @@ const content = "${loremIpsum}"
 	<body>
 		<h1>Astro</h1>
 		<div>
-			${Array.from({ length: 60 }).map(() => '<p>{content}</p>')}
+			${Array.from({ length: 100 }).map(() => '<p>{content}</p>').join('\n')}
+		</div>
+		<div>
+			${Array.from({ length: 50 }).map((_, i) => '<Paragraph num={' + i + '} str={content} />').join('\n')}
```

Comment:
Do we report these numbers somewhere? Like "Rendered 100 children and 50 astro components."

Reply:
We currently don't. I'm not sure we need to in case we update this file again, and we might forget updating the description in the other benchmark file.